best-practices/README.md

```diff
@@ -101,6 +101,10 @@ Rails
   environment variables are detected on deploy.
 * [Use blocks][date-block] when declaring date and time attributes in FactoryGirl factories.
 * Use `touch: true` when declaring `belongs_to` relationships.
+* Name service classes after the action they perform
```

Comment:
I think that distinction is key, especially given some of what I've seen on other recent client projects, where almost everything is put in `app/services`.

Unrelated to the naming/purpose of this, do we have any opinions on whether they should (or should not) allow for side-effects? Does the presence of side-effects impact what we name these?